src/services/itemDataService.ts

```
@@ -33,6 +33,7 @@
     IRegion
 } from "warframe-public-export-plus";
 import questCompletionItems from "@/static/fixed_responses/questCompletionRewards.json";
+import { IMessageDatabase } from "@/src/models/inboxModel";
```

ESLint

[error] 'IMessageDatabase' is defined but never used.

(@typescript-eslint/no-unused-vars)